### ISSUE TITLE
AlexNet Top-1 accuracy on test dataset only reach 45% when train with multiple GPUs

### ISSUE THREAD

**Issue by yiheng** [soumith/imagenet-multiGPU.torch#88]
Hi, Soumith

I train alexnet with 8 GPUs, and change the batchsize to 1024. After serval days, the top-1 accuracy on test dataset only get 45%, while on a single GPU it can get 55% after the same time. Are there any tricks when train with multiple GPU to get the same test accuracy?

Here's the command and summary:

th main.lua -data ./imagenet/ -nGPU 8 -nDonkeys 28 -batchSize 1024 -netType alexnet
[TRAINING SUMMARY] average loss (per batch): 1.82   accuracy(%):    top-1 55.99
[TESTING SUMMARY] average loss (per batch): 2.39   accuracy [Center](%):   top-1 46.43     

Thanks

**Comment by soumith** [comment]
@yiheng if you increase the batch size to something as large as 1024, the same learning rate / learning rate schedule might not apply. You will have to find different numbers for those parameters.

**Comment by yiheng** [comment]
Hi Soumith

Thanks for your suggestion. I'm trying with different lr / lr decay policy. The combination of the hyperparameters are too many and it takes a while to evaluate... Could you give some guide on how to tune these hypeparameters for a large batch? 

And a more basic question, **is a larger batch(such as 1024 or 2048) good or bad for minibatch-sgd training in deep learning?** Actually we're trying to scale out deep learning to multi-node and the batch size can't be too small or the communication time will domain... Look forward to your views :-)

Thanks

**Comment by soumith** [comment]
I've seen that
larger batch size ~= harder to find hyperparameters that will converge to the best accuracy.
